src/main/java/com/johan/blignaut/entelect/vitalityincubator/task2/helpers/GeoDBMappingHelper.java

```diff
@@ -0,0 +1,63 @@
+package com.johan.blignaut.entelect.vitalityincubator.task2.helpers;
+
+import com.johan.blignaut.entelect.vitalityincubator.restserver.model.GeoDBServer;
+import com.johan.blignaut.entelect.vitalityincubator.restclient.model.GeoDBClient;
+import com.johan.blignaut.entelect.vitalityincubator.restserver.model.MetaDataServer;
+import com.johan.blignaut.entelect.vitalityincubator.restserver.model.CityServer;
+import com.johan.blignaut.entelect.vitalityincubator.restclient.ApiException;
+import com.johan.blignaut.entelect.vitalityincubator.restclient.api.GeodbApi;
+import com.johan.blignaut.entelect.vitalityincubator.restclient.model.CityClient;
+import com.johan.blignaut.entelect.vitalityincubator.restserver.model.LinkServer;
+import com.johan.blignaut.entelect.vitalityincubator.restclient.model.LinkClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
```

Comment:
We don't compress our service call and mapping into a single class, we follow a single responsibility approach. 

So we'll have a service wrapper class that is invoked by the controller to fetch the 3rd party API data, then we take that model and give it to a mapper class and the mapper class does the mapping from 3rd party models to MSRV models. This way it will allow for independent testing of service wrapper and mapper classes if you're doing unit tests.